speedy-recovery-client/src/__tests__/components/SecuredMainPage.test.js

```diff
@@ -31,3 +31,11 @@ it("renders without crashing", () => {
     </MemoryRouter>
   );
 });
+
+test("SecuredMainPage renders without crashing", () => {
```

Comment:
How is this different from the test before?

Reply:
It can slightly increase test coverage

Reply:
But isn't it identical? What is the difference?

Reply:
```
 render() {
    if (!this.props.user) {
      return <Redirect to="/" />;
    }
```
It seems tests this.